crates/teloxide-core/src/types/story.rs

```diff
@@ -1,5 +1,81 @@
 use serde::{Deserialize, Serialize};
 
-/// TBA 6.8: currently it holds no information
+use crate::types::{Chat, ChatKind, StoryId};
+
+/// This object represents a story.
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-pub struct Story {}
+pub struct Story {
+    /// Chat that posted the story.
+    pub chat: Chat,
+
+    /// Unique identifier for the story in the chat.
+    pub id: StoryId,
```

Comment:
I think `id` should be at the top of the declaration, personally.

I know, telegram docs do exactly the opposite from time to time... :)